.circleci/config.yml

```diff
@@ -35,13 +59,26 @@ jobs:
           paths:
             - dist
 
+  test_python: # Python tests are separated from YAML tests, because the latter take longer. This would allow Python tests to use any other available container and yield it, once finished, to another job instead of counting only on the two containers assigned to the YAML tests.
+    docker:
+      - image: python:3.7
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          key: v1-py3-deps-{{ .Branch }}-{{ checksum "setup.py" }}
+
       - run:
-          name: Run tests
-          command: |
-            pytest `circleci tests glob "tests/**/[^_]*.py" | circleci tests split`
-            openfisca test `circleci tests glob "tests/**/*.{yaml,yml}" | circleci tests split`
+          name: Activate virtualenv
+          command: echo "source /tmp/venv/openfisca_france/bin/activate" >> $BASH_ENV
 
-  lint_yaml_files:
+      - run:
+          name: Run Python tests
+          command: pytest `circleci tests glob "tests/**/[^_]*.py" | circleci tests split`
```

Comment:
> Do we really have any gain from splitting the tests if there is no parallelism set up? 🤔

Reply:
You're right, I don't see any added value to the split.